File: fat_client/WPFUI/Models/JoinRoomFeedBack.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPFUI.Models
{
    class JoinRoomFeedBack
    {
        public Feedback feedback;
        public Room room_joined;
        public Boolean isPrivate;

        public JoinRoomFeedBack(Feedback fb, Room room_joined , Boolean isPrivate)
        {
            this.feedback = fb;
            this.room_joined = room_joined;
            this.isPrivate = isPrivate;
        }

    }
}

File: thin_client/app/src/main/java/com/example/thin_client/data/rooms/JoinRoomFeedback.kt
package com.example.thin_client.data.rooms

import com.example.thin_client.data.Feedback
import com.example.thin_client.data.model.Room

data class JoinRoomFeedback(
    val feedback: Feedback,
    val room_joined: Room?,
    val isPrivate: Boolean?
)